### DISCUSSION TITLE
Generate yaml template out of schema json

### DISCUSSION THREAD

**Discussion by flokr** [datamesh-architecture/datacontract-specification#104] [Q&A] [answered]
Hi everybody,

hopefully this is not a super-stupid question: is it possible to generate a template datacontract.yaml out of a datacontract.schema.json? Would be amazing to have such an option in the cli, like datacontract init --schema my.datacontract.schema.json

Thanks!

**Comment by jochenchrist** [comment, marked as answer]
Hi!

The current `datacontract init` uses the template https://datacontract.com/datacontract.init.yaml

You can specify a template (that you can maintain which follows your conventions and schema) using the `--template` property.

```
datacontract init --template my-custom-template.yaml
```

Hope this helps.

**Reply by flokr** [reply]
All clear, thank you!
I was hoping for a solution where I wouldn't have to manually keep the schema and the template in sync..so when changing the schema it would be possible to automagically generate a new template out of it. But anyways, thanks a lot! :)
